sim_nmlcc.py

```diff
@@ -59,32 +59,31 @@ def simulate(
     AXON = '"axon_group"'
     ALL = '"all"'
 
-    def mech(group, name, value, scal=1):
-        gmax = value*scal
-        decor.paint(group, arbor.density(name, dict(conductance=gmax)))
+    def mech(group, name, **kwargs):
+        decor.paint(group, arbor.density(name, dict(**kwargs)))
```

Comment:
So it seems like jNeuroML was doing something really strange with this code :)

```
    g = conductance  *  fopen ? evaluable
    gion = gmax * fopen 
    icl = gion * (v - ecl)
```

and then discarding `g`. Thanks for noticing